PocketSwift/PocketSwift/Packages/Core/Model/Transaction.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Transaction.swift
+//  PocketSwift
+//
+//  Created by Wilson Garcia on 3/23/19.
+//  Copyright © 2019 Wilson Garcia. All rights reserved.
+//
+
+import Foundation
+
+public struct Transaction: Model {
```

Comment:
we don't need the transaction model since what we send is the relay with the data json string. After signing the transaction we get a transaction object but we immediately retrieve the serializedTransaction property.

Reply:
Cool